content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+General
+=======
+
+**What is documented?**
+
+**For the Chevah project members:**
+
+* Packages, modules, classes and functions using docstrings. 
+
+* Documentation changes are linked to a certain version of SFTPPlus.  When 
+  changes are made, these are anchored to a specific release version.
+
+**For the customers:**
+
+* These can be pre-sales, trial customers and existing customers.
+
+* There are two main types of documentation - technical documentation and the
+  User's Guides section.
+
+* User interfaces that need further explanation and / or guidance.
+
+* User's Guides that are aimed at general audiences.
+  These require more in-depth detail than marketing / promotion materials on
+  the main website. 
+
+* Questions and issues to Support and Sales can instead be answered and
+  linked to using documentation.
+
+* Release notes that explains the changes tied to a specific version.
+
+* Further in-depth notes to help secure file transfer processes using SFTPPlus.
+
+For content that is more marketing / generalized, these will go in the main
+website content.
+
+
+Narrative documentation
+=======================
+
```

Comment:
I feel that this section is linked to "For the customers" but they way this document is organized is strange.

maybe put "For the customers" information as an introdcution to "narrative" documentation.

we have 2 main documentation types : code documentation vs narrative documentation

code documentation can be in the form of docstring, comments, examples, tests
narrative documentation can be technical docs / user guides / marketing / sales presentations